### PR TITLE
USWDS-Site: Transfer showcase content from USWDS repo and repair HTMLProofer error

### LINKED ISSUE #1990: USWDS-Site - Bug: HTML Proofer failing on legacy link
### Describe the bug

**When trying to create a [PR](https://github.com/uswds/uswds-site/pull/1982) from a branch off`main`, there is a failing step when running HTML proofer.**

For the Links > External check, the following failures were found:

* At ./_site/getting-started/showcase/all/index.html:2792:

  External link https://app.frontify.com/d/NZPXDvjOcz5x/nasa-web-style-guide failed (status code 404)

### Steps to reproduce the bug

1. Try to submit a PR against `main` 
2. Notice it fails on test_build at the HTML Proofer step 

### Expected Behavior

All external html links should be valid

### Screenshots

Screen Shot 2023-01-30 at 11 25 05 AM

### System setup

- Device: MacBook Pro
- OS: Monterey 12.6.2
- Browser: Chrome

### Additional context

> proof
> bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\://designsystem.digital.gov/:/' --ignore-files "//select-a-language//,//2017//,//2018//,//2019//" ./_site

Running 3 checks (Scripts, Links, Images) in ["./_site"] on *.html files...

Checking 892 external links
Checking 936 internal links
Checking internal link hashes in 202 files
Ran on 315 files!

For the Links > External check, the following failures were found:

* At ./_site/getting-started/showcase/all/index.html:2792:

  External link https://app.frontify.com/d/NZPXDvjOcz5x/nasa-web-style-guide failed (status code 404)

HTML-Proofer found 1 failure!

Exited with code exit status 1

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.

### PR DESCRIPTION
**Pull request by amyleadem**
## Summary
1. Transferred "Showcase" content from the `uswds` repo into `uswds-site` 
    - Related to issue https://github.com/uswds/uswds/issues/3324
1. Moved showcase content to `showcase.md` instead of `showcase-all.md`
1. Removed `showcase-all.md` and related references.
1. Removed references to `WHO_IS_USING_USWDS.md`
1. Removed the link to the  NASA Web Style Guide because it was returning a 404 and HTMLProofer error 
    - Related to #1990
    
The `uswds` repo update that removes the `WHO_IS_USING_USWDS.md` file can be found in PR https://github.com/uswds/uswds/pull/5120. 
:warning: This PR should be merged before merging https://github.com/uswds/uswds/pull/5120.

## Related issue
Closes #1990
Related to https://github.com/uswds/uswds/issues/3324

## Preview link
- [Showcase](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-move-showcase-info/documentation/showcase)
- [What's New](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-move-showcase-info/about/whats-new/)
- [Next](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-move-showcase-info/next/get-involved/)

## Solution
Issue #1990 revealed a 404 error with one of the links listed in `WHO_IS_USING_USWDS.md` in the `uswds` repo. 

Instead of repairing this issue in the `uswds` repo, this PR moves the content from `WHO_IS_USING_USWDS.md` directly into the showcase page and repairs the issue here. 

As described in https://github.com/uswds/uswds/issues/3324:
> There's no intrinsic relationship between the showcase data and USWDS source code, principles, guidance, etc. We should move this data to uswds-site to make it easier to update.

This PR also removes the unnecessary `showcase-all` subpage and updates all references to both `showcase/all` and `WHO_IS_USING_USWDS.md`.

## Testing and review
To review, confirm that:
1. All content appears as expected on the Showcase page.
1. Confirm that `npm run proof` returns no errors
1. Confirm that all references to `showcase/all` and `WHO_IS_USING_USWDS.md` have been appropriately updated.
1. Confirm that no suitable replacement exists for the NASA Web Style Guide link.

### CODE REVIEW

File: _next/05-get-involved.md [hunk 1]
```diff
@@ -64,15 +64,15 @@ chapter: true
 
 - Learn about [customization, theming,]({{ site.baseurl }}/documentation/settings/) [utilities]({{ site.baseurl }}/utilities/), and [tokens]({{ site.baseurl }}/design-tokens/) to adapt the design system to your agency’s brand
 
-- Review existing websites that use USWDS (such as those on the [designsystem.digital.gov Showcase]({{ site.baseurl }}/getting-started/showcase/all/)) to see how you could better integrate USWDS [principles]({{ site.baseurl }}/design-principles/) and [components]({{ site.baseurl }}/components/)
+- Review existing websites that use USWDS (such as those on the [designsystem.digital.gov Showcase]({{ site.baseurl }}/documentation/showcase/)) to see how you could better integrate USWDS [principles]({{ site.baseurl }}/design-principles/) and [components]({{ site.baseurl }}/components/)
```
Comment (mejiaj) [inline _next/05-get-involved.md:67]: # Optional

Seems odd that the design system url is mentioned after the abbreviated USWDS. Maybe we could simplify. Something along the lines of:

```suggestion
- Review existing websites that use USWDS (such as those in the [Showcase]({{ site.baseurl }}/documentation/showcase/)) to see how you could better integrate USWDS [principles]({{ site.baseurl }}/design-principles/) and [components]({{ site.baseurl }}/components/)
```

**Review by mejiaj — APPROVED** [review]
No issues found, I like this cleanup. Added two suggestions to consider moving forward. Either as changes now or follow-up enhancements.

File: pages/documentation/showcase.md [hunk 2]
```diff
@@ -3,11 +3,184 @@ permalink: /documentation/showcase/
...
+- [VA Caseflow](https://github.com/department-of-veterans-affairs/caseflow)
+- [VA Developer Portal](https://developer.va.gov/)
+- [VA Office of Information and Technology](https://www.oit.va.gov/)
+- [Virtual Student Foreign Service](http://vsfs.state.gov/)
+- [vote.gov](https://vote.gov/)
+- [Water Data for the Nation (USGS)](https://waterdata.usgs.gov/blog/)
+- [We the People (Petition the White House)](https://petitions.whitehouse.gov/)
+- [The White House](https://www.whitehouse.gov/)
+- [WIC Works Resource System - USDA](https://wicworks.fns.usda.gov/)
+- [Worker.gov](https://www.worker.gov/)
```
[the remark this answers is not in the document]
Comment (amyleadem) [reply pages/documentation/showcase.md:185]: Opened an issue for fast follow-up: #1992

**Review by mahoneycm — CHANGES_REQUESTED** [review]
@amyleadem  I found a potential replacement site for the nasa styleguide and mentioned it below!

I also found an old post that references `showcase/all`. The redirect works as expected but since you got rid of the others, I figured I'd flag

https://github.com/uswds/uswds-site/blob/58f19ef46a1a64ed6a917589a30869a48e24f768/_posts/2019-04-08-introducing-uswds-2-0.md?plain=1#L19

File: pages/documentation/showcase.md [hunk 3]
```diff
@@ -3,11 +3,184 @@ permalink: /documentation/showcase/
...
+- [LINCS - Adult Education and Literacy - U.S. Department of Education](https://lincs.ed.gov/)
+- [Legislative Branch Innovation Hub](https://usgpo.github.io/innovation/)
+- [Login.gov](https://www.login.gov/)
+- [Manufacturing USA](https://www.manufacturing.gov/)
+- [Medicaid](https://www.medicaid.gov/)
+- [Method Cards - 18F](https://methods.18f.gov/)
+- [Move.mil — Official DOD Moving Portal](https://www.move.mil/)
+- [MyMedicare](https://www.mymedicare.gov/)
+- [NASA Glenn Research Center](https://www1.grc.nasa.gov)
+- [NASA Human Computer Interaction Group](https://hci.arc.nasa.gov)
```
[the remark this answers is not in the document]
Comment (amyleadem) [reply pages/documentation/showcase.md:100]: @mahoneycm Thanks for digging into this. A good reminder to use the wayback! It looks like all the current style guide content is on GitHub, so there is not an active USWDS-based site to link to at this time. For that reason, I will continue leaving it off the list for now.

**Comment by amyleadem** [comment]
@mahoneycm good catch on the missed `showcase/all` link. Updated in https://github.com/uswds/uswds-site/pull/1991/commits/32dce0fd6b11a3371f7d248813c7d0ec581b401f

**Review by mahoneycm-gsa — APPROVED** [review]

**Review by mahoneycm — APPROVED** [review]

**Review by thisisdano — APPROVED** [review]
Good work!

**Pull request merged**